### PR TITLE
Update to speak.py - Windows 11

### PR DESCRIPTION
**Pull request by Exnav29**
I had an issue making the speak work on a Windows 11 PC running Python.  Updated this file so that it is compatible to run speak.py successfully.  Not 100% though as I do receive a "API request failed with status code: 404" in reference to the API url for Eleven Labs but the code runs and the speech is voiced.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thouroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: speech.mp3
Comment (nponeccop) [inline speech.mp3:1]: This file shouldn't be committed, should it?

**Review by nponeccop — CHANGES_REQUESTED** [review]

**Comment by nponeccop** [comment]
@Exnav29 There are conflicts now

**Comment by Exnav29** [comment]
Naturally there are as you delayed incorporating my change in favor of another one.  Just forget it, so glad I don't have to work with jerks like you every day.

**Comment by nponeccop** [comment]
Don't take me wrong. We merge 25+ PRs a day, in the order we encounter they are clean. Your PR is not labeled that means there are no objections. We didn't specifically prioritized it. You can look at successful PRs, there are like 5+ merges of conflicts on some. Just be patient, we have a long backlog and just randomly merge, preserving the rough order of batches (so once you are in a batch you get merged within 2 days if you cooperate on conflict resolution).

@p-i- @richbeales can you look whether it's a mergeable PR so that the author knows he is not wasting his time?

**Pull request closed without merging**